store/store.go

```diff
@@ -174,6 +177,27 @@ type Store struct {
 	suggestedCurrency string
 }
 
+func shouldRetryHttpResponse(resp *http.Response, attempt *retry.Attempt) bool {
+	return (resp.StatusCode == 500 || resp.StatusCode == 503) && attempt.More()
```

Comment:
I think we should only retry on 500 if we've hit the CDN, but that can be left for later lest this bitrots